components/Map/Map.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+
+import MapView, { Marker } from 'react-native-maps';
+import { StyleSheet, Text, View, Button, TextInput } from 'react-native';
+
+export default function Map() {
+
+let events = [
+    {
+      id: 1,
+      date: 20200407,
+      time: 1700,
+      unix_time: 1586278800,
+      activity: 'Volleyball',
+      location: 'Wash Park',
+      lat: 39.700,
+      long: -104.973,
+      duration: 'Could be a min and max input from the user -- ex. 15 min 120 max',
+      notes: 'If the weather is poor, meet here instead',
+      max_players: 10,
+      players_going: 5,
+      skill_level: 'Professional',
+      equipment: [
+        {
+          name: 'Net',
+          still_needed: false
+        },
+        {
+          name: 'Ball',
+          still_needed: true
+        }
+      ]
+    },
+    {
+      id: 2,
+      date: 20200409,
+      time: 1700,
+      unix_time: 1586451600,
+      activity: 'Spikeball',
+      location: 'Cheeseman Park',
+      lat: 39.731,
+      long: -104.973,
```

Comment:
Are we going to be using the longitude and latitude for pin placement on the map? I would like to discuss how this works with you and how we can best implement it for the BE.

Reply:
Yes that's how we will be placing pins on the map. Hopefully our BE can get us the coordinates when a user selects a location